<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DRS4RemoteControlServerConfigDlg</class>
 <widget class="QMainWindow" name="DRS4RemoteControlServerConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>95</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>268</width>
    <height>95</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>268</width>
    <height>95</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_autoLaunch">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: blue</string>
      </property>
      <property name="text">
       <string>launch automatically at program start ?</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_port">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
border-style: flat;
background-color: white; 
color: green;
border-radius: 10;
}

QPushButton:disabled {
border-style: flat;
background-color:  rgb(236,236,236); 
color: rgb(140,140,140);
border-radius: 10;
}

QPushButton:hover {
border-style: flat; 
background-color: rgb(216,216,216);
color: green;
border-radius: 5;
}

QPushButton:pressed {
border-style: flat; 
background-color: transparent;
color: green;
border-radius: 2;
border-color: black;
}</string>
        </property>
        <property name="text">
         <string>start listening ...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
